pkg/swagger/docs.go

```diff
@@ -1245,48 +1245,25 @@ const docTemplate = `{
                 }
             }
         },
-        "github_com_bacalhau-project_bacalhau_pkg_config_legacy_types.Duration": {
-            "type": "integer",
-            "enum": [
-                -9223372036854775808,
-                9223372036854775807,
-                1,
-                1000,
-                1000000,
-                1000000000,
-                60000000000,
-                3600000000000
-            ],
-            "x-enum-varnames": [
-                "minDuration",
-                "maxDuration",
-                "Nanosecond",
-                "Microsecond",
-                "Millisecond",
-                "Second",
-                "Minute",
-                "Hour"
-            ]
-        },
         "github_com_bacalhau-project_bacalhau_pkg_config_legacy_types.JobDefaults": {
             "type": "object",
             "properties": {
                 "executionTimeout": {
-                    "$ref": "#/definitions/github_com_bacalhau-project_bacalhau_pkg_config_legacy_types.Duration"
+                    "type": "integer"
```

Comment:
these are not integers but durations. The values can be 10s for seconds or 10h for hours. The new type is now at https://github.com/bacalhau-project/bacalhau/blob/1ca678c752371f790e84220e6061e63d3aed5e32/pkg/config/types/duration.go#L9

Reply:
https://cs.opensource.google/go/go/+/refs/tags/go1.23.1:src/time/time.go;l=632

Reply:
It is a limitation of our tool we use for swagger generation. https://github.com/swaggo/swag/issues/388

Reply:
But ideally 'Duration' can be casted to int64 as internally go uses the same.

Reply:
@wdbaruni  Let me know if this makes sense ? I can accordingly merge this ! Thanks !

Reply:
I understand. My problem is we are doing this change in the legacy config type instead of the one actually being used, and that we are defining the actual type of `types.Duration` in its usages instead of its definition or at least in a single place.

I have pushed a PR with these fixes that is based on this one https://github.com/bacalhau-project/bacalhau/pull/4552

If all looks good, feel free to merge it into this PR or directly into main